common/repositories/base/base_raid_members_repository.h

```diff
@@ -28,6 +28,7 @@ class BaseRaidMembersRepository {
 		int8_t      isgroupleader;
 		int8_t      israidleader;
 		int8_t      islooter;
+		int8_t			isbot;
```

Comment:
Looks like you made manual changes here - this file is code generated

See: https://docs.eqemu.io/server/database/repositories/#code-generation-repository-generator

Reply:
Good evening.  Yup, I updated manually.  :-)  I have now recreated using the auto-generator and pushed.  Also, after reviewing the Bots Enabled rule, was thinking that the proposed column (isbot) should be pushed to the non-bot database to streamline the raid code.  That is, just set it to 0 as default allowing the sql routines and other checks to be standardized and not duplicated for no specific purpose or cost.